WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -857,15 +865,33 @@ extension NotificationsViewController {
                 self.notificationCommentDetailCoordinator.onSelectedNoteChange = { [weak self] note in
                     self?.selectRow(for: note)
                 }
-
                 commentDetailViewController.navigationItem.largeTitleDisplayMode = .never
-                self.showDetailViewController(commentDetailViewController, sender: nil)
 
+                if shouldPushDetailsViewController {
+                    commentDetailViewController.hidesBottomBarWhenPushed = true
+                    navigationController?.pushViewController(commentDetailViewController, animated: true)
+                } else {
+                    self.showDetailViewController(commentDetailViewController, sender: nil)
+                }
```

Comment:
[Suggestion] I see this if statement duplicated throughout the file. How about abstracting this into a helper function that takes a view controller and displays it with a push or a showDetail?